committee-code-of-conduct/README.md

```diff
@@ -13,11 +13,11 @@ The Kubernetes Code of Conduct Committee (CoCC) is the body that is responsible
 
 ## Members
 
-* Jaice Singer Dumars (**[@jdumars](https://github.com/jdumars)**), Google
-* Paris Pittman (**[@parispittman](https://github.com/parispittman)**), Google
+* Jennifer Rondeau (**[@bradamant3](https://github.com/bradamant3)**), VMware
```

Comment:
non-blocking comment, can be moved to another PR: should be s/VMware/Stripe here.

Ref: https://github.com/kubernetes/community/pull/3523#discussion_r270214887